thetower/dtower/tourney_results/constants.py

```diff
@@ -75,7 +75,8 @@
 ][::-1]
 
 top = "Top"
-champ = "Champions"
+legend = "Legend"
+champ = "Champion"
```

Comment:
Can't rename this, it will cause issues with some folders and other shit. Don't even try